lib/order.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'menu'
+require_relative 'total'
+
+
+class Order
+  attr_reader :menu, :total
+
+  def initialize(menu = Menu.new, total = Total.new)
+    @menu = menu
+    @total = total
+  end
+
+  def add_to_order(dish, quantity)
+    menu.each do |item|
+      total.selection << item.merge({:quantity => quantity}) if item.value?(dish)

```

Comment:
I found it a bit hard to parse this line and figure out what the code is doing. You could consider refactoring it to make it more readable